hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -3153,5 +3199,42 @@
       stage: V2
       tests: ["chip_sw_pwrmgr_deep_sleep_all_wake_ups"]
     }
+
+    {
+      name: chip_sw_exit_test_locked
+      desc: '''End to end test to ensure rom boot strap can be performed after
+      exiting TEST_LOCKED.
+
+      - Pre-load the device into TEST_LOCKED and TEST_UNLOCKED states.
```

Comment:
I am a little bit lost here:
Are we looping this test twice: one with pre-load image `TEST_LOCKED` one with `TEST_UNLOCKED`?
I understand the part to preload `TEST_UNLOCKED` state,

but to pre-load `TEST_LOCKED` state, we still have to manually issue a LC transfer to go to `TEST_UNLOCKED` state, then we can advice to `PROD`, `PROD_END` right?

Reply:
I think the current definition is that we are able to use the exit token when we are in either the `LOCKED` or `UNLOCKED` state.

Can you have a look [here](https://docs.opentitan.org/doc/security/specs/device_life_cycle/#manufacturing-states)? You'll need to scroll down a bit for the table.  

It's possible the implementation diverged from this, so if that's the case we should probably just file an issue for future alignment but keep things as is for now.

Reply:
@cindychip i've updated this one to only check `TEST_UNLOCKED` states now, since it covers a separate scenario @alphan and i discussed. Actually given what he and I discussed, we can't really transition from `TEST_LOCKED` anyways, even if the spec allows it.